pySDC/helpers/plot_helper.py

```diff
@@ -2,6 +2,8 @@
 import matplotlib.pyplot as plt
 from distutils.spawn import find_executable
 
+default_mpl_params = mpl.rcParams.copy()
```

Comment:
This solves the problem where plots look different when generated by the CI pipeline, since the module is imported only once, so the first time plot_helper is imported, whatever are the settings then remain here until pytest is done. Ofc, if you change them and then import the helper, I guess this would mess things up, but this is the best I can do right now.